src/VirtoCommerce.ContentModule.Web/Controllers/Api/ContentController.cs

```diff
@@ -146,13 +146,16 @@ public async Task<ActionResult<ContentItem[]>> SearchContent(string contentType,
         {
             var storageProvider = _blobContentStorageProviderFactory.CreateProvider(GetContentBasePath(contentType, storeId));
 
-            var result = await storageProvider.SearchAsync(folderUrl, keyword);
-            var retVal = result.Results.OfType<BlobFolder>()
+            var searchResult = await storageProvider.SearchAsync(folderUrl, keyword);
+
+            var result = searchResult.Results.OfType<BlobFolder>()
                 .Select(x => x.ToContentModel())
                 .OfType<ContentItem>()
-                .Concat(result.Results.OfType<BlobInfo>().Select(x => x.ToContentModel()))
+                .Concat(searchResult.Results.OfType<BlobInfo>().Select(x => x.ToContentModel()))
+                .Where(x => !x.Name.EqualsInvariant("blogs"))
```

Comment:
Please make filtration only for top-level folders, inner folder should be able to contain "blogs" folder.